pkg/webhookadmission/utils.go

```diff
@@ -0,0 +1,141 @@
+package webhookadmission
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"time"
+
+	"github.com/portworx/sched-ops/k8s"
+	log "github.com/sirupsen/logrus"
+	hook "k8s.io/api/admissionregistration/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	webhookName        = "stork.webhook.com"
+	storkService       = "stork-service"
+	storkNamespace     = "kube-system"
```

Comment:
Stork can be started in any namespace, we should read that from an env variable.